hackathon/README.md

```diff
@@ -7,23 +7,106 @@ MDAnalysis core developers on code --- it can be your own project
+   [#hackathon channel][hackathon-channel]!
+ - Look through the [ideas below](#What-should-I-work-on?), and start
+   to think how you'd like use your time during the hackathon to
+   contribute to MDAnalysis.
+   - If you have a specific project of your own in mind: raise an issue
+     on the [UGM Issue Tracker][issue-tracker] so that you have a
+     dedicated place share and discuss your idea and gather other
+     attendees to help you. Also share your idea (with a link to the
+     issue) on the [#hackathon channel][hackathon-channel].
+   - If you see an existing idea (from the MDAnalysis team or another
+     attendee) that interests you, find the associated issue and join
+     the discussion there. Even if you don't intend to work on an
+     issue yourself (or are yet undecided), any questions, comments
+     and thoughts are always useful.
+
+### At the hackathon
+ - **Pick an idea and gather a team!** Use the associated issues (see
+   below) to find others who expressed interest in a particular project.
+   You can also team up with others to tackle multiple smaller issues
+   together.
+   
+   While you can choose to work individually, we strongly encourage you
+   to make use of this opportunity to work alongside like-minded
+   attendees this will both make the experience more fun, and improve
+   your code's quality.
+   - *Working on a project as a team:* Discuss how you could split your
+     project into different tasks. Make a central repository for your
+     code on GitHub, and use issues to keep track of and assign
+     developers to these tasks. Use Pull Requests (and push
+     frequently!) to share your progress with your team and allow them
+     to review or comment - and in turn, review and comment on your
+     teammates' progress. Eventually, these PRs can be merged to
+     combine everyone's work into a greater whole.
+     
+ - Update progress on the associated UGM issue (if one doesn't exist
+   yet, make one now) to let other attendees and the MDAnalysis team
+   know how you're going!
+ - Continue to use the [#hackathon][hackathon-channel] discord channel
+   for any broader discussion, and remember the MDAnalysis team is
+   on hand to help you out!
+ - Hack away, make new friends, build something awesome - and have fun!
+
+#### Showcase
+At the end of the session, we will be holding a brief showcase to allow
+those who wish to share the work they achieved over the hackathon. 
+
+This could be advertising a new feature you created or showing off a 
+bug fix you're particularly proud of; and need not be a "finished" 
+product - you can talk about what you got done, what future plans 
+are, and if there's still help wanted.
+
+If you do wish to share, add a slide for you/your team to the 
+shared presentation (*link TBA*) - a template will be provided. 
+Depending how many wish to present, time may be short, so have 
+an elevator pitch summary for your work ready!
```

Comment:
Details here are still vague; open to suggestions/comments. It would be nice to get an end-of-hackathon report from each group; hopefully this is a reasonable compromise between allowing attendees to say what they did without taking up too much of the limited time or being to logistically infeasible

Reply:
LGTM!